skimage/data/__init__.py

```diff
@@ -56,6 +57,7 @@ def camera():
     return load("camera.png")
 
 
+@deprecated()
```

Comment:
If this is called with a string argument, the user gets an even more helpful message about which function they should transition to. I'd recommend 

`@deprecated('skimage.data.astronaut()')`